packages/docs/package.json

```
@@ -23,6 +23,7 @@
 		"@cnamts/design-tokens": "^2.0.0",
 		"@cnamts/form-builder": "^2.0.0",
 		"@cnamts/vue-dot": "^2.0.0",
+		"@netlify/functions": "^0.7.2",
```

```suggestion
		"@netlify/functions": "0.7.2",
```
On fixe les versions (sauf les packages internes), c'est un bonne pratique apportée par l'utilisation du bot renovate
(pense à lancer un `yarn install` car ça affecte le `yarn.lock`